src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/utbetalingsoppdrag/UtbetalingsoppdragGenerator.kt

```diff
@@ -150,7 +302,8 @@ object UtbetalingsoppdragGenerator {
         return tilkjentYtelse?.let { ytelse ->
             ytelse.sisteAndelIKjede?.tilPeriodeId()
                 // TODO denne kan fjernes når den er patchet
-                ?: ytelse.andelerTilkjentYtelse.filter { it.periodeId != null }.maxByOrNull { it.periodeId!! }?.tilPeriodeId()
+                ?: ytelse.andelerTilkjentYtelse.filter { it.periodeId != null }.maxByOrNull { it.periodeId!! }
```

Comment:
TODO-en på linja over her - hva betyr det?

Reply:
Fjernet dette. Den ble patchet for lenge siden fabeef6f832f67e1774da60e6f049a2215aae666
I starten hadde vi ikke kontroll på "siste elementet i kjeden"